src/components/MyKiva/BorrowerCarousel.vue

```diff
@@ -82,6 +82,7 @@
 				<KvCarousel
 					ref="carousel"
 					class="borrower-carousel tw-w-full md:tw-overflow-visible"
+					:class="{'hidden-controls': filteredLoans.length < 1}"
```

Comment:
KvCarousel should automatically hide the controls when the slide count is 1: https://github.com/kiva/kv-ui-elements/blob/4b972972d46e123b4b7f82996b523bce9ee2d3a5/%40kiva/kv-components/src/vue/KvCarousel.vue#L35 Is that not working?

Reply:
Nice call! Thanks @eddieferrer!